doc/source/serve/tutorials/batch.md

```diff
@@ -186,4 +186,31 @@ Finally, let's run the script.
 $ python tutorial_batch.py
 ```
 
-You should get a similar output like before!
\ No newline at end of file
+You should get a similar output like before!
+
+## Troubleshooting
+
+If you see the following error:
+
+```console
+TypeError: Descriptors cannot not be created directly.
+    If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
+    If you cannot immediately regenerate your protos, some other possible workarounds are:   
+     1. Downgrade the protobuf package to 3.20.x or lower.
+     2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
+```
+
+You can downgrade the protobuf package to 3.20.x or lower by using a [runtime environment](runtime-environments).
```

Comment:
mention that they can also do this in their docker image

Reply:
Makes sense, added